openedx/core/djangoapps/course_date_signals/waffle.py

```diff
@@ -0,0 +1,24 @@
+"""
+This module contains various configuration settings via waffle switches for
+course date signals.
+"""
+
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+WAFFLE_FLAG_NAMESPACE = "course_date_signals"
+
+# .. toggle_name: course_date_signals.disable_spaced_out_sections
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to disable spaced out sections for self
+#   paced courses. When spaced out sections are enabled, graded content will be
+#   assigned a suggested relative due date, while attempting ot make sure that
+#   they are reasonably spaced out through out an ent. The algorithm attempts
+#   to suggest the relative due dates in such a way, that they are evenly
+#   spaced out through an estimated duration of a course.
```

Comment:
Could you expand on how the estimated duration of the course is calculated? If I remember correctly, it's retrieved from the Course Discovery, and there is some fallback when the value is missing there.

Reply:
Added a more detailed description.